02-peer-protocol.md

```diff
@@ -805,6 +813,8 @@ is destined, is described in [BOLT #4](04-onion-routing.md).
    * [`sha256`:`payment_hash`]
    * [`u32`:`cltv_expiry`]
    * [`1366*byte`:`onion_routing_packet`]
+   * [`u64`:`hold_fee_rate_day`]
```

Comment:
why not per block?

Reply:
I've chosen time because it matches the actual cost of locked funds closely. Blocks probably works just as well though.